Admin/Admin.php

```diff
@@ -2540,8 +2546,12 @@ public function id($entity)
     /**
      * {@inheritdoc}
      */
-    public function setValidator(ValidatorInterface $validator)
+    public function setValidator($validator)
```

Comment:
I dont understand you... How could you accept a different class if the interfaces is of another class?